src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.php

```diff
@@ -55,6 +55,7 @@
                 abstract_arg('senders service locator'),
             ])
         ->set('messenger.middleware.send_message', SendMessageMiddleware::class)
+            ->abstract()
```

Comment:
Intended?

Reply:
If I want to add arguments to this middleware, I need to add `abstract`.

https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php#L370

Reply:
Indeed.